### PR TITLE
Dem/review benchmarks

### PR DESCRIPTION
**Pull request by farrufat-cimne**
cancel folder name change

### CODE REVIEW

**Comment by farrufat-cimne** [comment]
@maceligueta

File: applications/DEMApplication/python_scripts/DEM_benchmarks_class.py [hunk 1]
```diff
@@ -3,8 +3,10 @@
 from KratosMultiphysics.DEMApplication import *
 import shutil
 from glob import glob
-from math import pi, sin, cos, tan, atan, fabs
+from math import pi, sin, cos, tan, atan, fabs, sqrt
```
Comment (GuillermoCasas) [inline applications/DEMApplication/python_scripts/DEM_benchmarks_class.py:6]: abs can be used, there is no need to import fabs.

File: applications/DEMApplication/python_scripts/DEM_benchmarks_class.py [hunk 2]
```diff
@@ -4203,11 +4213,828 @@ def compute_errors(self, velocity_list_outfile_name):  #FINALIZATION STEP
...
+
+                    force_node_x = node.GetSolutionStepValue(TOTAL_FORCES)[0]
+                    force_node_y = node.GetSolutionStepValue(TOTAL_FORCES)[1]
+                    force_node_z = node.GetSolutionStepValue(TOTAL_FORCES)[2]
+
+                    angular_node_x = node.GetSolutionStepValue(ANGULAR_VELOCITY)[0]
+                    angular_node_y = node.GetSolutionStepValue(ANGULAR_VELOCITY)[1]
+                    angular_node_z = node.GetSolutionStepValue(ANGULAR_VELOCITY)[2]
+
+                    displacement = [0]*3
```
Comment (GuillermoCasas) [inline applications/DEMApplication/python_scripts/DEM_benchmarks_class.py:4260]: you could use 
`def GetDisplacement(node):
     return node.X-node.X0, node.Y-node.Y0, node.Z-node.Z0`
here

File: applications/DEMApplication/python_scripts/DEM_benchmarks_class.py [hunk 3]
```diff
@@ -4203,11 +4213,828 @@ def compute_errors(self, velocity_list_outfile_name):  #FINALIZATION STEP
...
+                    displacement[0] = node.X - node.X0
+                    displacement[1] = node.Y - node.Y0
+                    displacement[2] = node.Z - node.Z0
+
+                    force_node = sqrt(force_node_x*force_node_x + force_node_y*force_node_y + force_node_z*force_node_z)
```
Comment (GuillermoCasas) [inline applications/DEMApplication/python_scripts/DEM_benchmarks_class.py:4316]: `force_node = sqrt(sum([node.GetSolutionStepValue(TOTAL_FORCES)[i]**2 for i in range(2)]))` and there is no need to define force_node_x, etc.

**Review by GuillermoCasas — COMMENTED** [review]
I added a few comments to illustrate how all this code can be improved. I am only commenting, since I understand there is no time for this now, but please take note.

**Comment by farrufat-cimne** [comment]
thx, 
these are good ideas but we will leave it for a future pull request since there are some bugs that require to be fixed and merged as soon as possible

**Review by maceligueta — APPROVED** [review]

**Pull request merged**